<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QLineEdit" name="lineEditOutFolder">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>751</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditImageFolder">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>751</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>I:\ProjektDermoSkopia\Proto4_MW\czw. sty 9 19_32_41 2020\tiff</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOpenImageFolder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Image Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOpenOutFolder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Out Folder</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>521</width>
      <height>370</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEditRegexImageFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>.+.tiff</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetImageFiles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditOut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>501</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>371</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBoxShowMatInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Mat Info</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show output</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxImRotate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Rotation</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show input</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelScaleBase">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale base</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxScaleBase">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>43</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="value">
      <number>6</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowTiffInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Tiff Info</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelScalePower">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale Power</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxScalePower">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>43</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxLoadAnydepth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Anydepth</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAutocleanOut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto clean output text</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowMask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show mask</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxTileSize">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>43</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>64</number>
     </property>
     <property name="maximum">
      <number>1024</number>
     </property>
     <property name="singleStep">
      <number>64</number>
     </property>
     <property name="value">
      <number>256</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelTileSize">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Tile size</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinTilePositionX">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>43</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinTilePositionY">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>43</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelTilePositionX">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Tile X</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTilePositionY">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Tile Y</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxTileStep">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>43</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>64</number>
     </property>
     <property name="maximum">
      <number>1024</number>
     </property>
     <property name="singleStep">
      <number>64</number>
     </property>
     <property name="value">
      <number>128</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelTileStep">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Tile step</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowTileOnImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Tile On Im</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxTileScale">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>43</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelTileScale">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Tile scale</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxShowMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="labelTransparency">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Transparency</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxTransparency">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>230</y>
       <width>43</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowMaskOnImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show mask on image</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="MyImageWidget" name="widgetImage" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>410</y>
      <width>151</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelPenSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Pen size</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxPenSize">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxConnected">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Connected</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFillHoles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Fill Holes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClearRegion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Mask on Tile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonReloadTile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Reload Tile Mask</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCopyToMask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy To Mask</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSaveMask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>SaveMask</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonReloadMask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Reload Mask</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyImageWidget</class>
   <extends>QWidget</extends>
   <header>myimagewidget.h</header>
   <container>1</container>
   <slots>
    <signal>on_mousePressed(QPoint,int)</signal>
    <signal>on_mouseMove(QPoint,int)</signal>
    <signal>on_KeyPressed(int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
